sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md

```diff
@@ -0,0 +1,236 @@
+
+// Extract text (OCR) from an image stream. This will be a synchronously (blocking) call.
+ImageAnalysisResult result = client.Analyze(
+    BinaryData.FromStream(stream),
+    VisualFeatures.Read);
+
+// Print text (OCR) analysis results to the console
+Console.WriteLine("Image analysis results:");
+Console.WriteLine(" Read:");
+
+foreach (DetectedTextBlock block in result.Read.Blocks)
+    foreach (DetectedTextLine line in block.Lines)
+    {
+        Console.WriteLine($"   Line: '{line.Text}', Bounding Polygon: [{string.Join(" ", line.BoundingPolygon)}]");
+        foreach (DetectedTextWord word in line.Words)
+        {
+            Console.WriteLine($"     Word: '{word.Text}', Confidence {word.Confidence.ToString("#.####")}, Bounding Polygon: [{string.Join(" ", word.BoundingPolygon)}]");
+        }
+    }
+```
+
+### Extract text from an image URL
+
+This example is similar to the above, expect it calls the `Analyze` method and provides a [publicly accessible image URL](https://aka.ms/azai/vision/image-analysis-sample.jpg) instead of a file name.
+
+```C# Snippet:ImageAnalysisExtractTextFromUrl
+// Extract text (OCR) from an image stream. This will be a synchronously (blocking) call.
+ImageAnalysisResult result = client.Analyze(
+    new Uri("https://aka.ms/azai/vision/image-analysis-sample.jpg"),
+    VisualFeatures.Read);
+
+// Print text (OCR) analysis results to the console
+Console.WriteLine("Image analysis results:");
+Console.WriteLine(" Read:");
+
+foreach (DetectedTextBlock block in result.Read.Blocks)
+    foreach (DetectedTextLine line in block.Lines)
+    {
+        Console.WriteLine($"   Line: '{line.Text}', Bounding Polygon: [{string.Join(" ", line.BoundingPolygon)}]");
+        foreach (DetectedTextWord word in line.Words)
+        {
+            Console.WriteLine($"     Word: '{word.Text}', Confidence {word.Confidence.ToString("#.####")}, Bounding Polygon: [{string.Join(" ", word.BoundingPolygon)}]");
+        }
+    }
+```
+## Troubleshooting
+
+### Common errors
+
+When you interact with Image Analysis using the .NET SDK, errors returned by the service correspond to the same HTTP status codes returned for REST API requests. For example, if you try to analyze an image that is not accessible due to a broken URL, a `400` error is returned, indicating a bad request.
+
+### Handling exceptions
+
+In the following snippet, the error is handled gracefully by catching the exception and displaying additional information about the error.
+
+```C# Snippet:ImageAnalysisException
+var imageUrl = new Uri("https://aka.ms.invalid/azai/vision/image-analysis-sample.jpg");
+
+try
+{
+    var result = client.Analyze(imageUrl, VisualFeatures.Caption);
+}
+catch (RequestFailedException e)
+{
+    if (e.Status == 400)
+    {
+        Console.WriteLine("Error analyzing image.");
+        Console.WriteLine("HTTP status code 400: The request is invalid or malformed.");
```

Comment:
would the `Message` property already contain an error similar to this? It seems odd to write your own 400 response error detail.